src/plugins/numberline/components/numberline-tile.tsx

```diff
@@ -222,19 +242,54 @@ export const NumberlineTile: React.FC<ITileProps> = observer(function Numberline
   svg.on("click", (e) => handleMouseClick(e));
   svg.on("mousemove", (e) => handleMouseMove(e));
 
-  // * =============================== [ Construct Numberline ] ================================ */
+  // * ================================ [ Construct Numberline ] =============================== */
+  const numOfTicks = 11;
+
+  //Returns an equally divided array between min and max with numOfTick # of elements
+  const generateTickValues = (min: number, max: number) => {
+    const tickValues = [];
+    const range = content.max - content.min;
+    for (let i = 0; i < numOfTicks; i++) {
+      const position = i / (numOfTicks - 1);
+      const tickValue = content.min + (position * range);
+      tickValues.push(tickValue);
+    }
+    return tickValues;
+  };
+
+  const tickFormatter = (value: number | { valueOf(): number }, index: number) => {
+    if (typeof value !== 'number') {
+      return value.toString();
+    }
+    if (value === content.min || value === content.max) {
+      return '';
+    }
+    return value.toFixed(1).toString();
```

Comment:
```suggestion
    return value.toFixed(1);
```
`toFixed()` already returns a string.